hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/HddsDatanodeService.java

```diff
@@ -660,8 +661,8 @@ private String reconfigBlockDeleteThreadMax(String value) {
   }
 
   private String reconfigDeletingServiceWorkers(String value) {
-    Preconditions.checkArgument(Integer.parseInt(value) >= 0,
-        OZONE_BLOCK_DELETING_SERVICE_WORKERS + " cannot be negative.");
+    Preconditions.assertTrue(Integer.parseInt(value) >= 0,
+        () -> OZONE_BLOCK_DELETING_SERVICE_WORKERS + " = " + value + " < 0");
```

Comment:
I think it's better to keep this patch limited to the `Preconditions.checkNotNull -> Objects.requireNonNull` migration.  Otherwise it becomes too big, with trivial and non-trivial changes mixed.